packages/@aws-cdk/cdk/lib/app.ts

```diff
@@ -15,6 +15,26 @@ export interface AppProps {
    */
   readonly autoRun?: boolean;
 
+  /**
+   * The output directory into which to emit synthesized artifacts.
+   *
+   * @default - If this value is _not_ set, considers the environment variable `CDK_OUTDIR`.
+   *            If `CDK_OUTDIR` is not defined, uses a temp directory.
+   */
+  readonly outdir?: string;
+
+  /**
+   * Include stack traces in construct metadata entries.
+   * @default true stack traces are included
+   */
+  readonly stackTraces?: boolean;
+
+  /**
+   * Include runtime versioning information in cloud assembly manifest
+   * @default true runtime info is included
```

Comment:
That is not true - because the CDK toolkit can override that though CLI arguments or environment variables, can't it?